### PR TITLE
Added real time visualization of shape creation and more improvements

### PR DESCRIPTION
**Pull request by VDykyi**
Added real time visualization of shape creation and more improvements

### CODE REVIEW

File: src/VAC/Global.h [hunk 1]
```diff
@@ -127,6 +127,23 @@ class Q_VPAINT_EXPORT Global: public QObject
     QColor faceColor();
 
     void setEdgeColor(const QColor& newColor);
+    void setFaceColor(const QColor& newColor);
+
+    bool drawShapeFaceEnabled() const;
```
Comment (marcus-sonestedt) [inline src/VAC/Global.h]: getters for bool props defining a single value should have "is" prefix according to Qt standard.

File: src/VAC/VectorAnimationComplex/VAC.cpp [hunk 2]
```diff
@@ -5814,25 +5819,67 @@ void VAC::updateCellsToConsiderForCutting()
...
+        if(emitSignal)
+        {
+            emit changed();
+        }
+    }
+}
+
+void VAC::addToHovered(const CellSet &cells, bool emitSignal)
+{
+    for(Cell * c: cells)
```
Comment (marcus-sonestedt) [inline src/VAC/VectorAnimationComplex/VAC.cpp]: "auto c" works too

**Review by marcus-sonestedt — APPROVED** [review]
Awesome work there, Vasyl! 👍👍

File: src/VAC/VectorAnimationComplex/VAC.cpp (hunk 2 above)
Comment (VDykyi) [reply src/VAC/VectorAnimationComplex/VAC.cpp]: Done

File: src/VAC/Global.h (hunk 1 above)
Comment (VDykyi) [reply src/VAC/Global.h]: Done

**Comment by VDykyi** [comment]
- Implemented the real time visualization when shape creation.
- Added ability to draw "faces"(fill) for closed shapes and ability to adjust it opacity and color.
- Added ability to adjusting of the highlighting and selecting colors and opacity for edges and fill due to current shape color.
- Also realized simplified selection of shapes.
- Added ability to show/hide the rectangle arround the shape during creation.

**Pull request merged**